tests/datasets/configuration/test_generic_metrics.py

```diff
@@ -1,6 +1,66 @@
+import pytest
+from jsonschema import validate
+from jsonschema.exceptions import ValidationError
 from yaml import safe_load
 
+from snuba.clickhouse.columns import AggregateFunction, Column, Float
+from snuba.datasets.configuration.json_schema import (
+    readable_storage_schema,
+    writable_storage_schema,
+)
+from snuba.datasets.configuration.utils import parse_columns
+from snuba.datasets.storages.generic_metrics import (
+    aggregate_common_columns,
+    bucket_columns,
+    common_columns,
+)
 
-def test_file_loads() -> None:
-    file = open("./snuba/datasets/configuration/generic_metrics/dataset.yaml")
-    safe_load(file)
+CONF_PATH = "./snuba/datasets/configuration/generic_metrics/storages"
+
+DISTRIBUTIONS_STORAGE_COLUMN_SET = [
+    *common_columns,
+    *aggregate_common_columns,
+    Column(
```

Comment:
Can we also import the aggregate functions instead of manually writing them out for the test?

Reply:
Eventually, would we get rid of our manual definitions of those columns since they'd be generated from config? In that case, we eventually remove the storage specific tests in favor of some generic testers for the JSON Schema?